maven/pom.xml

```diff
@@ -11,6 +11,10 @@
 	<packaging>pom</packaging>
 	<version>${revision}</version>
 
+	<prerequisites>
```

Comment:
We already use maven 3.6.3 (see mvnw) to build the maven plugins. This is also sort of wrong. I don't know what actual version of maven we need to _build_ the plugins. We don't test building against an older version of maven, so this prerequisite is not in anyway known to be true.

We do however set the version of maven to compile against

https://github.com/bndtools/bnd/blob/417811e3183520aa0977404b01aaa832bbf0ddf2/maven/pom.xml#L18

so that our plugins can be used with versions of maven back to 3.1.1. That is, we don't rely upon new maven API added since 3.1.1. It is our intention that the maven plugins can be used with older versions of maven.

It is not our intention to be able to build the maven plugins with older versions of maven. So this prerequisite is not accurate.